config/static/nodes/rpm-overrides/install-custom-rpm.yaml.envsubst

```diff
@@ -20,11 +20,15 @@ spec:
           [Unit]
           Description=Install custom runc
           # https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
-          After=network-online.target rpm-ostreed.service
+          After=network-online.target
+          # After=network-online.target
+          StartLimitIntervalSec=10
+          StartLimitBurst=30
 
           [Service]
-          Type=oneshot
+          Type=simple
```

Comment:
[nit] we can remove this line; `simple` is the default when specifying `ExecStart`.

Reply:
Thanks, removing and keeping it cleaner.